### PR TITLE
ISSUE-51: Change background segmentation method to selfie

### LINKED ISSUE #51: Change background segmenting method to SelfiSegmentation

### PR DESCRIPTION
**Pull request by jarkkokotaniemi**
closes #51

### CODE REVIEW

**Comment by vtiinanen** [comment]
image
This is the reason why the tests fail, I tried to mention it [earlier.](https://github.com/CumbersomePack/SmartCanvas/pull/61#discussion_r759568802)
Now there are actual tests in place to check such things and its currently failing.
https://uni-smartcanvas.herokuapp.com/ <- is also broken due this.

The fix should be simple :eagle: :earth_africa:

File: smart_canvas/core.py [hunk 1]
```diff
@@ -83,7 +83,8 @@ def enter(self, tick):
 
     def update(self, tick, frame):
```
[the remark this answers is not in the document]
Comment (Mikroudz) [reply smart_canvas/core.py:84]: See my commit to skip the startup.

File: smart_canvas/background.py [hunk 2]
```diff
@@ -1,20 +1,18 @@
 import cv2
 import numpy as np
-
+import mediapipe as mp
+mp_drawing = mp.solutions.drawing_utils
```
Comment (vtiinanen) [inline smart_canvas/background.py:4]: mp_drawing is not used anymore, right?

Comment (jarkkokotaniemi) [reply smart_canvas/background.py:4]: oh that's true

File: smart_canvas/background.py [hunk 3]
```diff
@@ -1,20 +1,18 @@
...
-        self.fg_mask = None
-        self.frame_list = []
-
-    def create_background(self, frame):
-        self.frame_list.append(frame)
+        self.selfie_segmentation = mp_selfie_segmentation.SelfieSegmentation(model_selection=1)
+        self.bg_image = None
+        self.output_image = None
 
     def fill_holes(self, mask):
```
Comment (vtiinanen) [inline smart_canvas/background.py:17]: Can we remove fill_holes?
There seems to be no usage of it.

Comment (jarkkokotaniemi) [reply smart_canvas/background.py:17]: I suppose, I left it there for if we want to keep in for the future but that is not part of the philosophy of this. Will be removed.

**Review by vtiinanen — APPROVED** [review]
👍

**Comment by jarkkokotaniemi** [comment]
Added the web-service fix to this branch as well

**Review by vtiinanen — APPROVED** [review]

**Review by Mikroudz — APPROVED** [review]

**Pull request merged**